opm/test_util/summaryComparator.cpp

```diff
@@ -45,6 +45,8 @@ SummaryComparator::SummaryComparator(const char* basename1, const char* basename
         this->keysShort = this->keys2;
         this->keysLong = this->keys1;
     }
+    
+    
```

Comment:
try to minimize the number of lines you touch.  Adding unnecessary whitespace changes confuses the reviewers.